src/rp2_common/pico_crt0/crt0.S

```diff
@@ -52,15 +52,26 @@ __vectors:
 #else
 .word isr_nmi
 .word isr_hardfault
+#if PICO_RP2040
 .word isr_invalid // Reserved, should never fire
 .word isr_invalid // Reserved, should never fire
 .word isr_invalid // Reserved, should never fire
 .word isr_invalid // Reserved, should never fire
+#else
+.word isr_memmanage
+.word isr_busfault
+.word isr_usagefault
+.word isr_securefault
+#endif
```

Comment:
Minor nitpick, but it kinda feels like `isr_memmanage` is a "specific" action, and `isr_invalid` is a "default" action, so I wonder if it might make more sense to write this as:
```c
#if PICO_RP2350
.word isr_memmanage
.word isr_busfault
.word isr_usagefault
.word isr_securefault
#else
.word isr_invalid // Reserved, should never fire
.word isr_invalid // Reserved, should never fire
.word isr_invalid // Reserved, should never fire
.word isr_invalid // Reserved, should never fire
#endif
```

? :shrug:

Reply:
convention is based on what is the more likely path in the future... we assume RP2040 is the minority